react-app/src/components/common/Table/Table.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import cn from "classnames";
+
+interface TableHeadProps {
+  className?: string;
+  children: React.ReactNode;
+}
+
+interface TableRowProps {
+  className?: string;
+  children: React.ReactNode;
+  isHeader?: boolean;
+}
+
+interface TableCellProps {
+  className?: string;
+  children: React.ReactNode;
+}
+
+interface TableProps {
+  className?: string;
+  children: React.ReactNode;
+}
+
+export const TableHead: React.FC<TableHeadProps> = ({
+  children,
+  className,
+}) => {
+  return <thead className={cn("bg-white", className)}>{children}</thead>;
+};
+
+export const TableRow: React.FC<TableRowProps> = ({
+  children,
+  className,
+  isHeader,
+}) => {
+  return (
+    <tr
+      className={cn(
+        !isHeader && "text-sm font-normal leading-5 text-gray-500",
+        className
+      )}
+    >
+      {children}
+    </tr>
+  );
+};
+
+export const TableCell: React.FC<TableCellProps> = ({
+  children,
+  className,
+}) => {
+  return <td className={cn("px-6", className)}>{children}</td>;
+};
+
+const Table: React.FC<TableProps> = ({ children, className }) => {
+  return (
+    <table className={cn("min-w-full rounded-lg shadow-md", className)}>
+      {children}
+    </table>
+  );
+};
+
+export default Table;
```

Comment:
I think it would be more suitable to either update SectionedTable to support your use case which i think can just be 
1. Add a flag to hide the section title 
2. Use SectionedTable with only one section

or the other way around which is the ideal case where we refactor SectionedTable to be a generic Table component and create another SectionedTable component that uses this generic Table component and injects the section headers. 

Otherwise we will need to manage loading and empty state as well as the sorting states everytime we integrate a table in a new screen